client/state/index.js

```diff
@@ -47,5 +47,8 @@ export function createReduxStore() {
 	) {
 		createStoreWithMiddleware = compose( createStoreWithMiddleware, window.devToolsExtension() );
 	}
-	return createStoreWithMiddleware( createStore )( reducer );
-};
+	if ( initialState === null ) {
+		initialState = {};
+	}
```

Comment:
In my testing default value only overrides undefined, not null. Try commenting this block out and running `make test` in `client/state`